$Thinkum$

Update make.sh to not overwrite any existing build-config

diff --git make.sh make.sh
index 8f8147fb3..f84dd588b 100755
--- make.sh
+++ make.sh
@@ -24,7 +24,7 @@ export LANG LC_ALL
 # thing" when run on the target machine, with the minor caveat that
 # any --xc-host parameter should be suitable for the host machine
 # instead of the target.
-sh make-config.sh "$@" || exit $?
+test -e output/build-config || { sh make-config.sh "$@" || exit $?; }
 
 . output/prefix.def
 . output/build-config
